k2hackbot/k2crawler/application_interaction/interaction.py

```diff
@@ -89,13 +94,28 @@ def get_element_to_interact_web_page(self):
                 element_list.append(element_obj)
         return element_list
 
+    def check_for_hidden_element(self, element):
```

Comment:
There is a hidden attribute similar to type which can also be used to hide any element from the screen please add check for that accordingly also return collectively

 ```suggestion
    def check_hidden_element(self, element)->bool:
    .... .... ...
    ...... ... ... 
    
    return is_element_hidden
```